src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$transitionTimeHeader: 300ms;
+$transitionTimeCard: 300ms;
```

Comment:
Why do you need 2 separate variables for the same value? 
```suggestion
$transition-duration: 300ms;
```

Reply:
in case we wanna have different durations in the future